README.md

```diff
@@ -1,2 +1,133 @@
-# notation-hashicorp-vault
-HashiCorp Vault provider for Notation CLI
+# notation-hc-vault
+HashiCorp Vault signing plugin for (Notation)[https://notaryproject.dev/]
+
+## Setup
+1. There are two binaries in the release: `key-helper` and `notation-hc-vault`.
+2. Install the `notation-hc-vault` plugin to the notation path specified by [notation plugin spec](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md#installation). On unix, the path is `$XDG_CONFIG_HOME/notation/plugins/hc-vault/notation-hc-vault`.
+3. `key-helper` is a standalone helper CLI used to configure Hashicorp Vault.
+
+## Install and Configure Hashicorp Vault Server
+1. [Install Hashicorp Vault](https://developer.hashicorp.com/vault/downloads)
+2. Create a vault config file under your work directory. A minimal working example is shown below, more details about config can be found [here](https://developer.hashicorp.com/vault/tutorials/operations/configure-vault#configuration-files)):
+    ```bash
+    $ cat > vault-server.hcl <<EOF
+    disable_mlock = true
+    ui            = true
+
+    listener "tcp" {
+    address     = "127.0.0.1:8200"
+    tls_disable = "true"
+    }
+
+    storage "file" {
+    path = "/tmp/vault-data"
+    }
+    EOF
+    ```
+3. Start a prod server 
+    ```bash 
+    vault server -config vault-server.hcl
+    ```
+   Note: when this process is stopped, the vault will be sealed. To unseal the vault, you need to run this command again.
```

Comment:
We typically refer to it as just "Vault" rather than "the vault":

```suggestion
   Note: when this process is stopped, Vault will be sealed. To unseal Vault, you need to run this command again.
```

> To unseal Vault, you need to run this command again.

I don't think this is strictly right either; Vault will be sealed once the process is terminated, but after re-starting this server process, you'll still need to unseal it:

https://developer.hashicorp.com/vault/docs/concepts/seal and https://developer.hashicorp.com/vault/docs/commands/operator/unseal 

```
$ vault operator unseal ... 
```

or via Web UI.

Reply:
(`the vault` versus `Vault` could be applied a bunch below :-)